sources/api/apiserver/src/server/controller.rs

```diff
@@ -374,6 +378,32 @@ where
     Ok(())
 }
 
+/// Dispatches an update command via `thar-be-updates`
+pub(crate) fn dispatch_update_command(args: Vec<String>) -> Result<HttpResponse> {
```

Comment:
```suggestion
pub(crate) fn dispatch_update_command(args: &[&str]) -> Result<HttpResponse> {
    let args = args.iter().map(ToString::to_string).collect();
```
This makes the calls friendlier, e.g. `dispatch_update_command(&["refresh"])`